_wiki/FAQ/General.md

```diff
@@ -2,63 +2,146 @@
 title: General FAQ
 ---
 
-### Application Data Directory {#appdata}
+### Is my tablet supported? {#supported-tablets}
 
-| Operating System | Path |
-| :--------------: | ---- |
-| Windows          | `%localappdata%\OpenTabletDriver`                  |
-| Linux            | `~/.config/OpenTabletDriver`                       |
-| macOS            | `~/Library/Application Support/OpenTabletDriver`   |
+Verify if your tablet is in the list of supported tablets [here]({% link _sections/Tablets.md %}). If it is not, you may do one of the following:
+
+- [Create a tablet support request in Github.](https://github.com/OpenTabletDriver/OpenTabletDriver/issues/new?assignees=&labels=configuration&projects=&template=tablet_configuration.yml&title=Add+support+for+)
+- [Create a tablet support thread in #config-creation channel of the Discord server.]({{ site.discord_invite_url }})
+- [Write the tablet configuration.]({% link _wiki/Development/AddingTabletSupport.md %})
```

Comment:
Not sure if this is actually enough to fix the classic "is Wacom Intuos supported?" question, but I think we can safely try this for now. We can always reintroduce it if the question gets too recurring.